pkg/plugin/processor/builtin/impl/field/convert_examples_test.go

```diff
@@ -144,3 +146,46 @@ func ExampleConvertProcessor_floatToString() {
 	//    }
 	//  }
 }
+
+//nolint:govet // a more descriptive example description
+func ExampleConvertProcessor_intTotime() {
+	p := NewConvertProcessor(log.Nop())
+
+	timeObj := time.Date(2024, 1, 2, 12, 34, 56, 123456789, time.UTC)
+
+	exampleutil.RunExample(p, exampleutil.Example{
+		Summary:     "Convert `string` to `time`",
```

Comment:
I guess this is a slip, should be int to time?

Reply:
Absolutely, good catch! 👀